schemas/common/external-source-system-audit.schema.json

```diff
@@ -0,0 +1,69 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/common/external-source-system-audit",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "External Source System Audit Attributes",
+  "type": "object",
+  "meta:extensible": true,
+  "description": "External Source System Audit fields.",
+  "definitions": {
+    "externalSourceSystemAudit": {
+      "properties": {
+        "xdm:createdBy": {
+          "title": "Created By",
+          "description": "Name of the user who created this record",
+          "type": "string"
+        },
+        "xdm:createdDate": {
```

Comment:
Why not use the created and modify date that all other schemas use - https://github.com/adobe/xdm/blob/master/schemas/external/repo/common.schema.json#L32